package.json

```diff
@@ -16,6 +16,7 @@
     "@babel/eslint-parser": "^7.14.7",
     "@babel/eslint-plugin": "^7.14.5",
     "@fontsource/inter": "^4.5.15",
+    "@hbsnow/rehype-sectionize": "^1.0.7",
```

Comment:
There is a more popular package `[rehype-section](https://www.npmjs.com/package/@agentofuser/rehype-section)`  but downloads aren't working for it.